modules/ROOT/pages/using-api-alerts.adoc

```diff
@@ -51,9 +47,13 @@ When you set an alert, you select users in your Business Group who can receive e
 +
 `Your API Version, jsonplaceholderapi - 1.0.development, is no longer in an alert state. The number of policy violations was not greater than 1 in the last 1 consecutive periods of 1 minutes.`
 
-After an alert fires, and API Manager sends the first set of two notification emails, and stops listening for alerts until the next alert period begins. This technique prevents repetitive notification emails.
+After an alert is triggered, API Manager sends the first set of two notification emails, and stops listening for alerts until the next alert period begins. This technique prevents repetitive notification emails.
+
+== Contracts Alerts
 
-== Alert Notification Categories
+API Manager automatically notifies users when actions related to contracts occur. For instance, when a client application requests access to an API instance, API Manager sends a notification with the status of the request. Depending on the configuration, status may indicate that the request is pending manual approval, or it may indicate that the request has been auto-approved.
```

Comment:
GLOBALLY replace "may" with wording that does not indicate permission. This is typically "might," although we try to avoid modal verbs (see style guide).

Reply:
Fixed